cdap-docs/admin-manual/source/operations/scaling-instances.rst

```diff
@@ -58,37 +69,43 @@ Scaling Services
 In a similar way to `Scaling Flowlets`_, you can query or change the number of handler instances of a Service
 by using the ``instances`` parameter with HTTP GET and PUT methods::
 
-  GET /v3/namespaces/default/apps/<app-id>/services/<service-id>/runnables/<runnable-id>/instances
-  PUT /v3/namespaces/default/apps/<app-id>/services/<service-id>/runnables/<runnable-id>/instances
+  GET /v3/namespaces/default/apps/<app-id>/services/<service-id>/instances
+  PUT /v3/namespaces/default/apps/<app-id>/services/<service-id>/instances
 
 with the arguments as a JSON string in the body::
 
   { "instances" : <quantity> }
 
 Where:
-  :<app-id>: Name of the application
-  :<service-id>: Name of the Service
-  :<runnable-id>: Name of the Service
-  :<quantity>: Number of handler instances requested
-  
-**Note:** In this release, the ``runnable-id`` is the same as the ``service-id``.
 
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Parameter
+     - Description
+   * - ``<app-id>``
+     - Name of the application
+   * - ``<service-id>``
+     - Name of the Service
+   * - ``<quantity>``
+     - Number of handler instances requested
+  
 Example: Find out the number of handler instances of the Service *RetrieveCounts*
 of the application *WordCount*::
 
-  GET /v3/namespaces/default/apps/WordCount/services/RetrieveCounts/runnables/RetrieveCounts/instances
+  GET /v3/namespaces/default/apps/WordCount/services/RetrieveCounts/instances
 
 Example: Change the number of handler instances of the Service *RetrieveCounts*
 of the application *WordCount*::
 
-  PUT /v3/namespaces/default/apps/WordCount/services/RetrieveCounts/runnables/RetrieveCounts/instances
+  PUT /v3/namespaces/default/apps/WordCount/services/RetrieveCounts/instances
 
 with the arguments as a JSON string in the body::
 
   { "instances" : 2 }
   
-Example using the :ref:`CDAP Standalone SDK<standalone-index>` and ``curl`` (reformatted to fit)::
+Example using the :ref:`CDAP Standalone SDK <standalone-index>` and ``curl`` (reformatted to fit)::
 
-  curl -vw'\n' -XPUT 'http://localhost:10000/v3/namespaces/default/apps/WordCount/services/RetrieveCounts/runnables/RetrieveCounts/instances' \
+  curl -w'\n' -X PUT 'http://localhost:10000/v3/namespaces/default/apps/WordCount/services/RetrieveCounts/instances' \
```

Comment:
If you copy and paste the curl call, it adds a trailing return, and so that instead of the results returned being mangled by your command line, it gives a nice clean result. It makes your next command line prompt appear on the next line, and makes the results match the output given in the example exactly. Yes, we could drop them, but I though that having curl calls that can be cleanly cut, pasted, and get clear results was worth the extra bit of code.